content/client-server-api/modules/end_to_end_encryption.md

```diff
@@ -597,6 +597,30 @@ package naming convention.
 
 {{% event event="m.key.verification.cancel" %}}
 
+###### General key verification error handling
```

Comment:
It might be good to link to this section from the section above, where it says "Verifications can be cancelled by either device at any time...".